dadjokes.go

```diff
@@ -26,7 +27,7 @@ func httpRequest() string {
 	req, err := http.NewRequest("GET", url, nil) // crafting HTTP request with url and userAgent
 	if err != nil { // if err, loggging error via Zerolog and returning error to chat
 		sublogger.Warn().Err(err).Msg("HTTP request formatting error")
-		return "An error occurred: " + string(err)
+		return "An error occurred: " + string(err.Error())

```

Comment:
```suggestion
              return fmt.Sprintf("An error occured: %v", err)
```

Reply:
err.Error() already returns a string, it's useless to cast it's return to a string.

Reply:
Oh, I didn't know that!

Reply:
addressed by 10243c2